src/components/inputtext/InputText.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const InputText = ({label, placeholder}) => {
+  return(
+    <div className="input-group">
+      <label className="input-label">{label}</label>
+      <input type="text" className="input-text-field" placeholder={placeholder}/>
```

Comment:
this can be named as `input__text-field`

Reply:
🆗